tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
```

Comment:
Oh whoops wait, I don't have access to this branch. But you know what, I'm happy with it; testing only in Node is actually fine because it will still catch the error if we regress, so I'm happy with this. Thanks for the PR!!